vcx/libvcx/src/settings.rs

```diff
@@ -187,9 +185,20 @@ pub fn test_agency_mode_enabled() -> bool {
 }
 
 pub fn process_config_string(config: &str) -> Result<u32, u32> {
+    use utils::logger::LibvcxDefaultLogger;
+    use serde_json::Value;
     let configuration: Value = serde_json::from_str(config).or(Err(error::INVALID_JSON.code_num))?;
     if let Value::Object(ref map) = configuration {
         for (key, value) in map {
+            if key == DEFAULT_LOGGING_KEY {
```

Comment:
Can't we just use the RUST_LOG env variable for the default logger?  Seems easy and gets rid of this workaround.

Reply:
yes, working on this change.